accounts/views.py

```diff
@@ -43,7 +43,7 @@
 COUNTDOWN_SECONDS = login_expiration_seconds + (60 * 15)
 
 logger = logging.getLogger(__name__)
-ACL_GOOGLE_GROUP = settings.OPEN_ACL_GOOGLE_GROUP
+# ACL_GOOGLE_GROUP = settings.OPEN_ACL_GOOGLE_GROUP
```

Comment:
I assume this is a merge issue. Or I probably comment it out because it throws errors on starting the application.